File: Homeworks/C#1/03.OperatorsAndExpressionsHomework/05.ThirdDigitIsSeven/ThirdDigitIsSeven.cs
using System;

class ThirdDigitIsSeven
{
    static void Main()
    {
        Console.Write("Please, enter number: ");
        int number = int.Parse(Console.ReadLine());

        bool check = (number / 100) % 10 == 7;
        Console.WriteLine("The third digit (right-to-left) of " + number + " is 7 ===> " + check);
    }
}

File: Homeworks/JavaScriptFundamentals/02.OperatorsAndExpressions/04.ThirdDigitIsSeven/ThirdDigitIsSeven.js
/*
 Write an expression that checks for given integer if
 its third digit (right-to-left) is 7. E. g. 1732 ? true.
 */

var jsConsole;
var button = document.getElementById('check');

button.onclick = checkNumber;

function checkNumber() {
    var input = jsConsole.readInteger('#input');
    var thirdDigit = Math.floor(input / 100) % 10;
    var isSeven = thirdDigit === 7;

    jsConsole.writeLine('Third digit of number ' + input + ' is 7: ' + isSeven);
    console.log('Third digit of number ' + input + ' is 7: ' + isSeven);
}